src/ucp/core/ucp_ep.c

```diff
@@ -334,7 +334,9 @@ ucs_status_t ucp_ep_init_create_wireup(ucp_ep_h ep, unsigned ep_init_flags,
     }
 
     ep->am_lane = key.am_lane;
-    ep->flags  |= UCP_EP_FLAG_CONNECT_REQ_QUEUED;
+    if (ucp_ep_get_cm_lane(ep) == UCP_NULL_LANE) {
```

Comment:
reuse https://github.com/openucx/ucx/pull/5220/files#diff-6ec7d6c877f4e5aae320aa49bd08f22cR241

Reply:
ok, will create another PR (as you sugested) and reuse it in other ones